app/templates/components/event-card.hbs

```diff
@@ -15,17 +15,21 @@
       </a>
     {{/unless}}
     <div class="main content">
-      <a class="header" href="{{href-to 'public' event.identifier}}">
-        <span>{{event.name}}</span>
-      </a>
+      {{#smart-overflow}}
+        {{#link-to 'public' event.identifier class='ui header'}}
+          <span>{{event.name}}</span>
+        {{/link-to}}
+      {{/smart-overflow}}
       <div class="meta">
         <span class="date">
           {{moment-format event.startTime 'ddd, MMM DD HH:mm A'}}
         </span>
       </div>
-      <div class="description">
-        {{event.shortLocationName}}
-      </div>
+      {{#smart-overflow}}
```

Comment:
Make it such that all that needs to be done here is 

```hbs
{{#smart-overflow class='description' }} 
	{{event.shortLocationName}}
{{/smart-overflow}}
```

Putting a span inside should be taken care of by the component.